doc/source/installation.rst

```diff
@@ -69,6 +69,22 @@ some benefits:
 * allows molecule installation at user level without even needing to have
   the script in ``PATH``.
 
+Docker
+======
+
+We publish molecule images via `quay.io`_ where the following tags are available:
+
+  * ``latest``: latest master branch build, which should be viewed as unstable
```

Comment:
same comment as above

Reply:
Not currently possible, hence suggesting people use named versions. We will look at fixing this in 2.21